setup.py

```diff
@@ -1,7 +1,5 @@
 import os
-import sys
 from setuptools import setup
-from setuptools.command.test import test as TestCommand
```

Comment:
These are left-overs from #51 and we should merge these changes even if the rest is not.